e2e/quit-on-close.e2e.spec.ts

```diff
@@ -1,28 +1,28 @@
 import { test, expect, ElectronApplication } from '@playwright/test';
 
-import { createDefaultSettings, packageLogs, startRancherDesktop, teardown } from './utils/TestUtils';
+import { createDefaultSettings, reportAsset, startRancherDesktop, teardown } from './utils/TestUtils';
 
 /**
  * Using test.describe.serial make the test execute step by step, as described on each `test()` order
  * Playwright executes test in parallel by default and it will not work for our app backend loading process.
  * */
 test.describe.serial('quitOnClose setting', () => {
-  test.afterAll(async() => {
-    await packageLogs(__filename);
-  });
-
   test('should quit when quitOnClose is true and window is closed', async() => {
+    const logName = `${ __filename }-quitOnCloseTrue`;
     createDefaultSettings({ application: { window: { quitOnClose: true } } });
-    const electronApp = await startRancherDesktop(__filename, { tracing: false });
+    const electronApp = await startRancherDesktop(__filename, { logName });
 
     await electronApp.firstWindow();
 
     await expect(closeWindowsAndCheckQuit(electronApp)).resolves.toBe(true);
+    await electronApp.context().tracing.stop({ path: reportAsset(logName, 'trace') });
```

Comment:
why dont we have `reportAsset` being used to trace the logs in the following test case `should not quit when quitOnClose is false and window is closed` ?

Reply:
Because the normal `teardown()` call handles that for us. We don't call `teardown()` in this case.